.github/workflows/test.yml

```diff
@@ -7,20 +7,16 @@ on:
 jobs:
   lint:
     runs-on: ubuntu-20.04
```

Comment:
Is there a reason the test workflow runs on `20.04` but the release & publish workflows run on `latest`?

Reply:
The tests need to run on LTS-1 because that's last supported, the workflows that don't test / build native extensions run on latest because it's the default, but they could easily run on 20.04 for safety.